.github/workflows/triage-unallowed-contributions.yml

```diff
@@ -20,8 +22,17 @@ jobs:
               ...context.repo,
               commit_sha: context.sha
             })
+            const pullNumber = pulls.data
+              .map(pull => pull.number)
+              .shift()
 
-            return pulls.data.map(pull => pull.number).shift()
+            if (pullNumber) {
+              console.log(`Pull request number: ${pullNumber}`)
+              return pullNumber
+            } else {
+              console.log(`When this workflow ran, the associated pull request was not yet created. Pushing a new commit after you've created a pull request will automatically re-run this workflow, or you can manually re-run the workflow.`)
+              process.exit(1)
```

Comment:
@rachmar Will this result in all squashes/merges to `main` resulting in a failed check?

Perhaps the `push` event for this workflow should be excluded from running when on `main`?